web/themes/new_weather_theme/templates/partials/daily-forecast-quick-toggle.html.twig

```diff
@@ -0,0 +1,84 @@
+{#
+This template expects two variables available to it:
+- content: The full object created from the daily forecast block
+- dayIndex: The index of the day currently being rendered within
+the content
+#}
+{% set dayId = day.periods[0].monthAndDay | lower | replace({" ":"-"}) %}
+{% set hasAlertIcon = day.alerts.metadata.highest != "" %}
+{% set temps = day.periods | map(p => p.data.temperature.degF) %}
+{% set low = min(temps) %}
+{% set high = max(temps) %}
+{% set pop = day.maxPop %}
+{% if pop is null %}
+{% set pop = 0 %}
+{% endif %}
+<button
+  class="wx-quick-toggle-item bg-white flex-align-center display-flex flex-wrap width-full padding-105"
+  id="{{dayId}}-quick-forcast-button"
+  role="button"
+  aria-controls="daily-content-{{itemId}}"
+  aria-expanded="true"
+>
+  <div class="wx-quick-toggle-item-top display-flex flex-1 flex-justify flex-align-center text-primary-darker font-mono-sm">
+    <time class="text-align-left flex-1 text-primary-dark" datetime="{{day.periods[0].startTime}}" aria-label="{{"Date" | t}}">
```

Comment:
I think the aria-label here may need to be the date value instead of "Date" because otherwise the SR repeats "Date" for every item in the quick forecast, and doesn't announce the actual value

<img width="654" alt="Screenshot of voice over text" src="https://github.com/user-attachments/assets/9ccbf08d-73d7-44f2-986b-35c8b2fb11a1">